dolphinscheduler-dist/src/main/assembly/dolphinscheduler-bin.xml

```diff
@@ -107,6 +107,21 @@
             <directoryMode>0755</directoryMode>
         </fileSet>
 
+        <fileSet>
+            <directory>${basedir}/../config</directory>
+            <outputDirectory>conf</outputDirectory>
+        </fileSet>
+
+        <fileSet>
+            <directory>${basedir}/.././</directory>
+            <includes>
+                <include>mvnw</include>
+                <include>mvnw.cmd</include>
```

Comment:
The binary zip will not contain maven wrapper directory, can this work?

Reply:
I read the script in mvnw seems this will dowload a maven binary.

Reply:
> I read the logic in mvnw seems this will dowload a maven binary.

Yes.